Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```
@@ -1683,14 +1683,219 @@ script:
       required: true
     description: Retrieves attachments from a ticket.
     name: servicenow-get-ticket-attachments
-  dockerimage: demisto/python3:3.11.10.116949
+  - arguments:
+    - description: 'Short description of the ticket. Use issue fields as variables to enrich the text. Example: “${alert.name}”'
+      name: short_description
+      prettyname: Short Description
+      required: true
+    - auto: PREDEFINED
+      defaultValue: incident
+      description: Ticket type. Can be "incident", "problem", "change_request", "sc_request", "sc_task", "sc_req_item", "sn_si_incident" or "std_change_template". Default is "incident".
+      name: ticket_type
+      prettyname: Ticket Type
+      predefined:
+        - incident
+        - problem
+        - change_request
+        - sc_request
+        - sc_task
+        - sc_req_item
+        - sn_si_incident
+        - std_change_template
+      required: true
+    - auto: PREDEFINED
+      description: Ticket urgency. You can either select from the predefined options or enter another value. For example, "Urgent" or "5".
+      name: urgency
+      predefined:
+        - 1 - High
+        - 2 - Medium
+        - 3 - Low
+    - auto: PREDEFINED
+      description: Ticket severity. You can select from the predefined options.
+      name: severity
+      prettyname: Severity
+      required: true
+      predefined:
+        - 1 - High
+        - 2 - Medium
+        - 3 - Low
+    - auto: PREDEFINED
+      description: Ticket impact.
+      name: impact
+      predefined:
+        - 1 - High
+        - 2 - Medium
+        - 3 - Low
+    - auto: PREDEFINED
+      description: Whether to set the ticket as Active. Can be "true" or "false".
+      name: active
+      predefined:
+        - 'true'
+        - 'false'
+    - description: The ticket activity due date, in the format "2016-07-02 21:51:11".
+      name: activity_due
+    - description: List of users assigned to the ticket.
+      name: additional_assignee_list
+    - description: User assigned to the ticket.
+      name: assigned_to
+    - description: Category of the ticket.
+      name: category
+    - description: UID Format.
```

```suggestion
    - description: UID format.
```
I don't understand this description